src/util/helpers.tsx

```diff
@@ -298,3 +299,15 @@ export const getFileExtension = (filename: string): string => {
 
   return `.${filename.split(".").pop()}` || "";
 };
+
+export const getDevices = (
+  instance: LxdInstance,
+): [string, LxdDeviceValue][] => {
+  const instanceDevices = Object.entries(
+    instance.expanded_devices ?? {},
+  ).filter(([, device]) => {
+    return !(device.type === "nic" || device.type === "disk");
```

Comment:
```suggestion
    return (device.type !== "nic" && device.type !== "disk");
```
I think this is easier to read.